crates/driver/src/arguments.rs

```diff
@@ -185,8 +192,13 @@ impl std::fmt::Display for Arguments {
         writeln!(f, "node_url: {}", self.node_url)?;
         writeln!(f, "http_timeout: {:?}", self.http_timeout)?;
         writeln!(f, "use_internal_buffers: {}", self.use_internal_buffers)?;
-        write!(f, "transaction_submission_nodes: ",)?;
+        write!(f, "transaction_submission_nodes: ")?;
         display_list(self.transaction_submission_nodes.iter(), f)?;
+        writeln!(
+            f,
+            "fee_objective_scaling_factor: {}",
+            self.fee_objective_scaling_factor,
+        )?;
         writeln!(f)?;
```

Comment:
I believe `writeln!(f);` has to move up to end the line of the `transaction_submission_nodes` argument before printing the new argument.

```suggestion
        writeln!(f)?;
        writeln!(
            f,
            "fee_objective_scaling_factor: {}",
            self.fee_objective_scaling_factor,
        )?;
```

Reply:
🤦 - nice catch!